tests/no-side-effects-in-initialization.test.ts

```diff
@@ -1,10 +1,10 @@
-/* Possible improvements for rollup:
- * * Properly handle valid this values
- * * Deleting an object expression member whose value is a global does not need to be a side-effect
+/** Possible improvements for rollup:
+ * Properly handle valid this values
+ * Deleting an object expression member whose value is a global does not need to be a side-effect
```

Comment:
This is actually supposed to be a bulleted list. If you want to improve its looks, maybe change it to

```suggestion
/* Possible improvements for rollup:
 * - Properly handle valid this values
 * - Deleting an object expression member whose value is a global does not need to be a side-effect
```

Reply:
Yes, it's my misunderstanding.

Reply:
Yes, it's my misunderstanding.